Packages/Timeline/Sources/Timeline/TimelineView.swift

```diff
@@ -75,6 +75,7 @@ public struct TimelineView: View {
       if viewModel.client == nil {
         viewModel.client = client
         viewModel.timeline = timeline
+        viewModel.account = account
```

Comment:
You don't need to pass the Current account env here. You can directly use `CurrentAccount.shared`, it's fine as a singleton.

Reply:
Missed the singleton. Thanks!